src/form.js

```diff
@@ -18,27 +18,38 @@ export default class Form extends React.Component {
       question: newCardData.get('question'),
       answer: newCardData.get('answer')
     }
+    event.target.reset()
+    if (this.props.mode === 'edit') {
```

Comment:
Instead of relying on a `mode` prop, you can check to see if the card you received as a prop already has an `id` property. See my note about passing the right card instead of the whole array.